source/protocol/lsp-messages.ads

```diff
@@ -360,7 +360,7 @@ package LSP.Messages is
    --  }
 
    type AlsReferenceKind is
-     (Simple, Write, Static_Call, Dispatching_Call, Parent, Child);
+     (Simple, Access_Ref, Write, Static_Call, Dispatching_Call, Parent, Child);
```

Comment:
Could you please update comments above ( `reference_kinds ALS extension:`) and doc/reference_kinds.md file?

Reply:
Done, the naming is a little different because "access" is a keyword in Ada.